src/rpc/methods/eth/eth_tx.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright 2019-2024 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use super::*;
+use crate::eth::EthTx;
+
+/// Ethereum Improvement Proposals 1559 transaction type. This EIP changed Ethereum’s fee market mechanism.
+/// Transaction type can have 3 distinct values:
+/// - 0 for legacy transactions
+/// - 1 for transactions introduced in EIP-2930
+/// - 2 for transactions introduced in EIP-1559
+pub const EIP_LEGACY_TX_TYPE: u64 = 0;
+pub const EIP_1559_TX_TYPE: u64 = 2;
+
+pub const ETH_LEGACY_HOMESTEAD_TX_CHAIN_ID: u64 = 0;
```

Comment:
Let's have these constants in the `eth` module.

Reply:
Fixed.